src/main/scala/sbtdatabricks/DatabricksHttp.scala

```diff
@@ -142,6 +142,148 @@ class DatabricksHttp(endpoint: String, client: HttpClient, outputStream: PrintSt
   }
 
   /**
+   * Create an execution context
+   * @param language the relevant coding language
+   * @param cluster the relevant cluster within which the context will be created
+   * @return The id of the execution context
+   *
+   */
+  private[sbtdatabricks] def createContext(language: DBCExecutionLanguage, cluster: Cluster): ContextId = {
+    outputStream.println(s"Creating '${language.is}' execution context on cluster '${cluster.name}'")
+    val post = new HttpPost(endpoint + CONTEXT_CREATE)
+    val request = CreateContextRequestV1(language.is, cluster.id)
+    val form = new StringEntity(mapper.writeValueAsString(request))
+    form.setContentType("application/json")
+    post.setEntity(form)
+    val response = client.execute(post)
+    val handler = new BasicResponseHandler()
+    val responseString = handler.handleResponse(response).trim
+    outputStream.println(s"Creating Context: Received the following response: '${responseString}'")
+    mapper.readValue[ContextId](responseString)
+  }
+
+  /**
+   * Check status of an execution context
+   * @param contextId Contains the id of the execution context
+   * @param cluster the relevant cluster
+   * @return status of the execution context
+   */
+  private[sbtdatabricks] def checkContext(contextId: ContextId, cluster: Cluster): ContextStatus = {
+    outputStream.println(s"Checking execution context '${contextId.id}' on cluster '${cluster.name}'")
+    val form =
+      URLEncodedUtils.format(List(new BasicNameValuePair("clusterId", cluster.id),
+                                  new BasicNameValuePair("contextId", contextId.id)), "utf-8")
+    val request = new HttpGet(endpoint + CONTEXT_STATUS + "?" + form)
+    val response = client.execute(request)
+    val handler = new BasicResponseHandler()
+    val responseString = handler.handleResponse(response).trim
+    outputStream.println(s"Checking Context: Received the following response: '${responseString}'")
+    mapper.readValue[ContextStatus](responseString)
+  }
```

Comment:
we can add a status update here. It might be nice to override the `toString` methods of `CommandStatus` and `CommandResults` to display a more human readable update. If it's pending, that's all we need to show. If an error has occurred, let's show the cause. If all worked out, let's show the result.